src/Hintman/Suggestion/Aggregation.hs

```diff
@@ -0,0 +1,28 @@
+module Hintman.Suggestion.Aggregation
```

Comment:
Heh, now looking at this over time I really feel that the top-module documentation is missing here..